integration-tests/mybatis/pom.xml

```diff
@@ -59,6 +59,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-h2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+        </dependency>
```

Comment:
I admit I am coming a bit late to the party, but anyway, I'd vote for keeping awaitility in the test scope. It is better to keep the compile dependencies minimal, especially because of the native compilation.

Reply:
Thanks @ppalaga - I will raise a follow issue to fix it.